view/frontend/web/js/model/checkout.js

```diff
@@ -33,6 +33,16 @@ function(
      */
     hasDeliveryOptions: ko.observable(false),
 
+    /**
+     * The country code.
+     */
+    countryId: ko.observable(null),
+
+    /**
+     * Best package type.
+     */
+    bestPackageType: 'package',
```

Comment:
Kun je kijken of het werkt zonder default value hier? Ik wil eigenlijk niet dat dit hier moet staan

Reply:
done